### PR TITLE
expand_as

### PR DESCRIPTION
**Pull request by bilal-aamer**
Closes https://github.com/unifyai/ivy/issues/21925, Closes https://github.com/unifyai/ivy/issues/21974

### CODE REVIEW

**Comment by bilal-aamer** [comment]
@zaeemansari70 Any pointers to address the failing test cases, would be appreciated.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # global
 from hypothesis import strategies as st
+from hypothesis import draw
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:3]: This import should be removed from here

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Requested some changes, let me know if you have any questions. Thanks!

**Comment by bilal-aamer** [comment]
@zaeemansari70 kindly review

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 2]
```diff
@@ -399,6 +400,50 @@ def test_paddle_expand(
...
+
+
+@st.composite
+def dtype_x_and_shape(draw):
+    input_dtype = draw(st.sampled_from(["float32", "int32", "float64"]))
+    shape = draw(st.lists(st.integers(min_value=1, max_value=10), min_size=1, max_size=4))
+    x = draw(helpers.array_values(shape=shape, dtype=input_dtype))
+    return input_dtype, x, shape
+
+@given(dtype_x_and_shape())
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:430]: the decorator used here should be `@handle_frontend_test`, kindly refer to our docs for a better understanding.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]

**Comment by aparajith21** [comment]
Hi, please let me know when everything @zaeemansari70 suggested has been incorporated and this is ready for review!

**Comment by bilal-aamer** [comment]
Sure @aparajith21 👍

**Review by bilal-aamer — COMMENTED** [review]
@aparajith21 Kindly check it out 

@zaeemansari70 👀

**Comment by aparajith21** [comment]
Hey, thanks for contributing! Could you please only add the function you're creating and not modify any of the rest of the file please? Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by bilal-aamer** [comment]
@aparajith21 @Fayad-Alman Could you let me know how to proceed further.

**Comment by NripeshN** [comment]
Hey @bilal-aamer,
Could you please resolve the merge conflicts so it is easier for the reviewer to review

**Comment by Ishticode** [comment]
closing this as too many changes (besides the issues) like it adds `test_paddle_abs` in paddles manipulations tests which already exist https://github.com/unifyai/ivy/blob/2c11f9f2bf9070bef18a10ee91d7e66a44fdaca3/ivy_tests/test_ivy/test_frontends/test_paddle/test_manipulation.py#L307

**Pull request closed without merging**